fe/fe-core/src/main/java/com/starrocks/sql/analyzer/FunctionAnalyzer.java

```diff
@@ -193,6 +193,26 @@ private static void analyzeBuiltinAggFunction(FunctionCallExpr functionCallExpr)
                 throw new SemanticException("window argument must be numerical type");
             }
         }
+        
+        if (fnName.getFunction().equalsIgnoreCase(FunctionSet.MAX_BY)) {
```

Comment:
equals is enough. because  all function name is lowercase after sql parser.

Reply:
fixed